lib/core/src/server/utils/load-custom-babel-config.js

```diff
@@ -46,15 +46,20 @@ function loadFromPath(babelConfigPath) {
   return config;
 }
 
-function isBabelLoader8() {
+function isBabelLoader8(options) {
+  if (options.defaultConfigName === 'create-react-app') {
```

Comment:
`defaultConfigName` was removed for the v5, because it has no value. Can you please explain what is the purpose of this condition?

Reply:
This doesn't seem to be needed at all, as npm/Yarn hoist `babel-loader`.